onnxruntime/core/session/inference_session.h

```diff
@@ -20,6 +20,13 @@ namespace ONNX_NAMESPACE {
 class ModelProto;
 }  // namespace ONNX_NAMESPACE
 
+struct OrtCustomOpDomain {
+  std::string domain_;
+  int op_version_start_{};
```

Comment:
as we discussed and agreed, for custom ops in custom domain, let's have their version always to be "1", since it's really hard to maintain a version history for custom ops in custom domains (with custom op libs). In this way, the versioning is always [1, 1].

Reply:
Yep, I just didn't make the change yet since our existing custom op test model used version 5.

Reply:
disable the test please given we don't have enough time to fix the test for the release.

Reply:
Yep, doable, I addressed Scott & Pranav's feedback and will submit soon!